compiler/release

```diff
@@ -76,6 +76,15 @@ sync; sync; sync;
 rm "$native_bin"/*.txt
 sync; sync; sync;
 
+# Testing of the native binaries
+mv "$directory/bin" "$directory/bin.bak"
```

Comment:
Let's add `rm -rf bin.bak` here. Otherwise the build will fail if bin.bak exists already.